flume/conf/agent.conf.template

```diff
@@ -15,7 +15,7 @@
 # For those usages not covered by the GNU Affero General Public License please contact with iot_support at tid dot es
 
 #=============================================
-# To be put in APACHE_FLUME_HOME/conf/cygnus.conf
```

Comment:
The CHANGES_NEXT_RELEASE entry is missing, although maybe in this case is not needed if I understanding correctly and thi PR fix some problem introduce in a very recent PR.

Just a thought... mark this as NTC if it doesn't make sense to add entry to CHANGES_NEXT_RELEASE.